test/runtests.jl

```diff
@@ -3,9 +3,6 @@ using Test
 using Statistics
 using Random
 using LinearAlgebra
-
-using ZigZagBoomerang
-
 using ZigZagBoomerang: poisson_time
```

Comment:
```suggestion
using ZigZagBoomerang
using ZigZagBoomerang: poisson_time
```
Let's keep the package, otherwise only `poisson_time` is exported